### DISCUSSION TITLE
Blender Geometry Nodes & Godot

### DISCUSSION THREAD

**Discussion by IsaacDdR** [godotengine/godot-proposals#4048] [General Discussions]
Having Blender's Geometry Nodes inputs inside Godot would be great for procedural generation.
UE4 has a plugin called Altermesh for it and works great.

**Comment by eon-s** [comment]
The idea is cool but Altermesh cost $99, so it may not be something easy to reproduce on an engine.

If you have an idea of how to make it, it may be possible to propose something.

**Comment by mrbbbaixue** [comment]
check this out: protongraph/protongraph

Although there's already a addon for this, but I think it can become a cool core feature.

**Reply by Ophiolith** [reply]
Your link is broken ;) [ProtonGraph](https://github.com/protongraph/protongraph) is here. It does look extremely cool.

**Reply by Zireael07** [reply]
Another idea: https://github.com/setzer22/blackjack (Early development, Godot integration planned)

**Comment by ultramarinebicycle** [comment]
>  UE4 has a plugin called Altermesh for it and works great.

And Houdini bridge. Although I'm not sure if geometry nodes are mature enough to have bridge software for yet. Hopefully someone with more experience with them can chime in.

**Comment by azur-wolve** [comment]
Such thing would be great, but it won't be core; a plugin is the way to bring it to life.
